src/tools/assign-related-messages-to-package.js

```diff
@@ -1,10 +0,0 @@
-module.exports = (nsp, pkgs) => pkg => {
```

Comment:
this file is still being referred to in [assign-related-messages-to-package.spec.js](https://github.com/matthias-reis/rlsr/blob/master/spec/assign-related-messages-to-package.spec.js#L4) I assume that the spec-file is outdated and can be deleted? @matthias-reis